### DISCUSSION TITLE
Channel type must be voice or stage, got GUILD_VOICE!

### DISCUSSION THREAD

**Discussion by ustokan** [Androz2091/discord-player#629] [General]
I get this error while using Queue.connect()

**Screenshots**
![image](https://user-images.githubusercontent.com/55207603/126704275-3f6125f9-e2b2-4767-89e0-a6ce43c9a084.png)

 - Node Version: [14.17.3]
 - Discord Player Version: [5.0.0-dev.399105e.1626955320]
 - Discord.js Version: [13.0.0-dev.5e70694.1626955541]

**Comment by twlite** [comment]
update to the latest version:

```sh
$ npm r discord-player
$ npm i discord-player@dev
```

**Comment by ustokan** [comment]
Error has not disappeared
![image](https://user-images.githubusercontent.com/55207603/126782277-36d09126-0ae1-4b5a-864a-7047ebd405cb.png)
![image](https://user-images.githubusercontent.com/55207603/126782310-2d59c8e0-d978-453b-9bfc-e93895b15f01.png)

**Comment by twlite** [comment]
if you inspect the **[source code](https://unpkg.com/discord-player@5.0.0-dev.399105e.1626955320/lib/Structures/Queue.js)**, the code actually has `GUILD_VOICE` or `GUILD_STAGE_VOICE`. The problem is likely with your installation. Try removing `node_modules` and re-install

**Reply by ustokan** [reply]
Fixed, thanks
